clash-ghc/src-ghc/Clash/GHC/ClashFlags.hs

```diff
@@ -60,6 +60,8 @@ flagsClash r = [
   , defFlag "fclash-hdlsyn"                  $ SepArg (setHdlSyn r)
   , defFlag "fclash-nocache"                 $ NoArg (liftEwM (setNoCache r))
   , defFlag "fclash-noclean"                 $ NoArg (liftEwM (setNoClean r))
+  , defFlag "fclash-no-prim-warn"            $ NoArg (liftEwM (setNoPrimWarn r))
+  , defFlag "fclash-no-color"                $ NoArg (liftEwM (setNoColor r))
```

Comment:
Would be cool if could hook into GHC's [-fdiagnostics-color](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/flags.html#ghc-flag--fdiagnostics-color=%E2%9F%A8always|auto|never%E2%9F%A9)

Reply:
Also we should rename the the others to `-fclash-no-cache` and `-fclash-no-clean` to be consistent with the new flags and with all of [GHC's flags](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/flags.html).

Reply:
Agreed to both. I'll create a separate issue for the flag renaming, as I think it's too invasive to handle in this PR. I'll look into `-fdiagnostic-color`, thanks!